src/libraries/System.Collections.Immutable/src/System/Linq/ImmutableArrayExtensions.cs

```diff
@@ -358,8 +358,7 @@ public static T First<T>(this ImmutableArray<T> immutableArray)
 
             // In the event of an empty array, generate the same exception
             // that the linq extension method would.
-            return immutableArray.Length > 0
-                ? immutableArray[0]
+            return !immutableArray.IsEmpty ? immutableArray[0]
```

Comment:
Did the fixer make this change?  It appears to be getting the whitespace "wrong"; we should validate that trivia is being handled correctly.

Reply:
Yes, this was made by the fixer, I assume the expected is to keep the new line, right?

Reply:
Yes

Reply:
Right.  In general we want to keep all "trivia" the same, modifying as little as is necessary to apply the fix.